src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,52 @@
 package pl.javastart.task;
 
-public class Main {
+import java.util.*;
 
+public class Main {
     public static void main(String[] args) {
 
+        String filePath = "src/main/java/resources/vehicles.csv";
+        VehiclesWorkshop vehiclesWorkshop = new VehiclesWorkshop(filePath);
+        Scanner scan = new Scanner(System.in);
+        boolean continueProcess = false;
+        do {
+            int number = getNumber(scan);
+            switch (number) {
+                case 0 -> {
+                    continueProcess = false;
+                    if (!vehiclesWorkshop.getVehicles().isEmpty()) {
+                        FileOperations.writeQueueToFile(filePath, vehiclesWorkshop.getVehicles());
```

Comment:
Gdy kolejka będzie pusta, to też chcemy ją zapisać do pliku. Jeżeli tego nie zrobisz, to przy kolejnym uruchomieniu programu będziesz miał w kolejce te same pojazdy, które już obsłużyłeś